vendor/knative.dev/serving/pkg/apis/config/defaults.go

```diff
@@ -137,6 +136,15 @@ func NewDefaultsConfigFromMap(data map[string]string) (*Defaults, error) {
 		return nil, err
 	}
 
+	// We default this to what the user has specified
+	nc.RevisionRequestStartTimeoutSeconds = nc.RevisionTimeoutSeconds
```

Comment:
I see this PR: https://github.com/knative/serving/pull/13256

for the actual update.

Reply:
With out this vendor change, we have non matching yaml manifest, and a broken operator releaes. Hence approving....

/cc @houshengbo